google/cloud/storage/idempotency_policy_test.cc

```diff
@@ -516,6 +516,51 @@ TEST(StrictIdempotencyPolicyTest, PatchDefaultObjectAclIfMatchEtag) {
   EXPECT_TRUE(policy.IsIdempotent(request));
 }
 
+TEST(StrictIdempotencyPolicyTest, GetProjectServiceAccount) {
+  StrictIdempotencyPolicy policy;
+  internal::GetProjectServiceAccountRequest request(
+      "test-project-id");
+  EXPECT_TRUE(policy.IsIdempotent(request));
+}
+
+TEST(StrictIdempotencyPolicyTest, ListNotification) {
+  StrictIdempotencyPolicy policy;
+  internal::ListNotificationsRequest request("test-bucket-name");
+  EXPECT_TRUE(policy.IsIdempotent(request));
+}
+
+TEST(StrictIdempotencyPolicyTest, CreateNotification) {
+  StrictIdempotencyPolicy policy;
+  internal::CreateNotificationRequest request("test-bucket-name", NotificationMetadata());
+  EXPECT_FALSE(policy.IsIdempotent(request));
+}
+
+TEST(StrictIdempotencyPolicyTest, CreateNotificationIfMatchEtag) {
```

Comment:
I can't find in the docs where `If-Match` applies to NotificationConfig creation, and after trying it via a curl command in my terminal, it seems to have no effect.

If you found something that says otherwise in the docs, please link it -- I couldn't find anything.

Reply:
This:

https://cloud.google.com/storage/docs/json_api/v1/parameters

says:

> "those parameters that apply to all Google Cloud Storage JSON API operations are shown below."

But if it does not work, then it does not work and we should fix this.  I need to run, but maybe you can ask folks how this *should* work?

Reply:
Done.